web-ui/src/main/resources/catalog/locales/fr-core.json

```diff
@@ -480,5 +480,9 @@
     "warnPublishDraft": "Lors de la publication d'une fiche ayant un statut brouillon, le statut passera à approuvé. Voulez-vous continuer ?",
     "cancelWorkingCopy": "Annuler la copie de travail",
     "deleteWorkingCopyRecordConfirm": "Voulez-vous vraiment supprimer la copie de travail ?",
-    "workingCopy": "Brouillon"
-}
\ No newline at end of file
+    "workingCopy": "Brouillon",
+    "moreFacet": "de plus",
+    "lessFacet": "de moins",
+    "possibleValue": "valeurs possibles",
+    "only": "seulement"
```

Comment:
Non-english translations should not be committed manually as they are stored in Transifex and will cause conflicts when importing updated translation files automatically.